test/llvm-intrinsics/fpclass.ll

```diff
@@ -0,0 +1,451 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+
+; Just check that reverse translation works
```

Comment:
Can we have a quick check to ensure that fpclass is not generated here?

Thanks

Reply:
I guess it's being checked implicitly by the fact, that forward translation doesn't error out due to unknown intrinsic